enhancements/compact-clusters.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Compact Clusters
+authors:
+  - "@smarterclayton"
+reviewers:
+  - "@derekwaynecarr"
+  - "@IronCladLou"
+  - "@crawford"
+  - "@hexfusion"
+approvers:
+  - "@derekwaynecarr"
+creation-date: "2019-09-26"
+last-updated: "2019-09-26"
+status: implementable
+see-also:
+replaces:
+superseded-by:
+---
+
+# Compact Clusters
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Open Questions [optional]
+
+This is where to call out areas of the design that require closure before deciding
+to implement the design.  For instance, 
+ > 1. This requires exposing previously private resources which contain sensitive
+  information.  Can we do this? 
+
+## Summary
+
+OpenShift is intended to run in a wide variety of environments. Over the years
+we have refined and reduced the default required footprint - first by removing
+the hard requirement for infrastructure nodes in 4.1, and second by preparing
+for 3-node clusters in 4.2 with the introduction of the masterSchedulable flag.
+OpenShift should continue to take steps to reduce the default footprint and
+incentivize improvements that allow it to work with smaller 3-node and eventually
+one node clusters.
```

Comment:
Should this enhancement also address reducing the compute resource requirements of the cluster? [k3s](https://k3s.io/) for example.